tests/e2e/testcases/subnet/sov/etna/suite.go

```diff
@@ -25,7 +25,7 @@ const (
 	testLocalNodeName = "e2eSubnetTest-local-node"
 )
 
-func createEtnaSubnetEvmConfig() {
+func createEtnaSubnetEvmConfigPOA() {
```

Comment:
can we just pass variable to the `createEtnaSubnetEvmConfig()` instead so we can chose between "--proof-of-authority" and "--proof-of-stake"  so we can keep single func to create blockchain config VS having 2 very similar func( 1 line difference)

Reply:
another option is to use `CreateEtnaSubnetEvmConfig` from `tests/e2e/commands/etna.go` 
I think it already supports both PoS and PoA usecase